packages/block-editor/src/components/block-list/use-block-props/use-zoom-out-mode-exit.js

```diff
@@ -30,7 +32,26 @@ export function useZoomOutModeExit( { editorMode } ) {
 				return;
 			}
 
+			const sectionsClientIds = getBlockOrder( getSectionRootClientId() );
+
+			const isSectionBlock = sectionsClientIds.includes( clientId );
```

Comment:
I guess in the future this could use the `isSectionBlock` selector, but it seems incompatible with zoom out mode right now because it checks for navigation mode.

Reply:
That selector should be mode agnostic surely? I'll take a look at that.

Reply:
I guess the selector is saying that there are no section blocks when in a mode that doesn't support sections.

When looked at that way zoom out could be included in part of the selector's logic.

The selector does also include a few other clauses, so would need to also check they don't introduce edge cases for zoom out.